### PR TITLE
Issue #23 Created PassportEntity class

### LINKED ISSUE #23: Create PassportEntity
- Create PassportEntity.java in Package entities.

- Create class variables for every column.

- Determine required variables for this entity.

- Entity should contain constructors(all args, required args), getters, setters.

- You should override equals and hashcode methods.

You can read about it at: https://www.baeldung.com/java-equals-hashcode-contracts

### PR DESCRIPTION
**Pull request by ChirkinaTanya**

### CODE REVIEW

**Review by nikaalin — APPROVED** [review]

**Review by a-deriabin — APPROVED** [review]

**Review by JustRevolt — APPROVED** [review]

File: src/main/java/com/ifmo/epampractice/entities/PassportEntity.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
+package com.ifmo.epampractice.entities;
+
+import java.time.LocalDate;
+import java.util.Objects;
```
Comment (PavelEzhkov) [inline src/main/java/com/ifmo/epampractice/entities/PassportEntity.java:4]: Remove this unused import

File: src/main/java/com/ifmo/epampractice/entities/PassportEntity.java [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+                issuer.equals(that.issuer) &&
+                issueDate.equals(that.issueDate) &&
+                expirationDate.equals(that.expirationDate) &&
+                serialNumber.equals(that.serialNumber);
+    }
+
+    @Override
+    public int hashCode() {
+        int result = id;
+        result = 31 * result + issueCountry.hashCode();
```
Comment (PavelEzhkov) [inline src/main/java/com/ifmo/epampractice/entities/PassportEntity.java:87]: what happens if issueCountry is null?

File: src/main/java/com/ifmo/epampractice/entities/PassportEntity.java [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+        return serialNumber;
+    }
+
+    public void setSerialNumber(String serialNumber) {
+        this.serialNumber = serialNumber;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
```
Comment (PavelEzhkov) [inline src/main/java/com/ifmo/epampractice/entities/PassportEntity.java:73]: Where is braces {} ?
https://www.oracle.com/technetwork/java/codeconventions-150003.pdf 12 page

File: src/main/java/com/ifmo/epampractice/entities/PassportEntity.java [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+
+public class PassportEntity {
+    private int id;
+    private String issueCountry;
+    private String issuer;
+    private LocalDate issueDate;
+    private LocalDate expirationDate;
+    private String serialNumber;
+
+    public PassportEntity(int id, String issueCountry, String issuer, LocalDate issueDate, LocalDate expirationDate, String serialNumber) {
```
Comment (PavelEzhkov) [inline src/main/java/com/ifmo/epampractice/entities/PassportEntity.java:14]: line is too long, split it

**Review by PavelEzhkov — COMMENTED** [review]
Fix it

File: src/main/java/com/ifmo/epampractice/entities/PassportEntity.java (hunk 1 above)
Comment (ChirkinaTanya) [reply src/main/java/com/ifmo/epampractice/entities/PassportEntity.java:4]: fixed

File: src/main/java/com/ifmo/epampractice/entities/PassportEntity.java (hunk 2 above)
Comment (ChirkinaTanya) [reply src/main/java/com/ifmo/epampractice/entities/PassportEntity.java:87]: fixed

File: src/main/java/com/ifmo/epampractice/entities/PassportEntity.java (hunk 3 above)
Comment (ChirkinaTanya) [reply src/main/java/com/ifmo/epampractice/entities/PassportEntity.java:73]: fixed

File: src/main/java/com/ifmo/epampractice/entities/PassportEntity.java (hunk 4 above)
Comment (ChirkinaTanya) [reply src/main/java/com/ifmo/epampractice/entities/PassportEntity.java:14]: fixed

**Pull request closed without merging**